deepchem/models/tensorgraph/layers.py

```diff
@@ -7,7 +7,7 @@
 import tensorflow as tf
 import numpy as np
 
-from deepchem.models.tensorgraph import model_ops, initializations, regularizers, activations
+from deepchem.nn import model_ops, initializations, regularizers, activations
```

Comment:
deepchem.nn has been removed recently so you might be on an old version of DeepChem